rio/src/RIO/Prelude.hs

```diff
@@ -443,3 +444,6 @@ import qualified Data.Text.Encoding.Error (lenientDecode)
 
 import qualified Control.Monad.Primitive (primitive)
 import qualified Control.Monad.ST
+
+aempty :: Control.Applicative.Alternative f => f a
```

Comment:
Please include a minor version bump, a Haddock comment explaining what this identifier is, a `@since` comment, and a ChangeLog entry.

Reply:
For sure - commit incoming.
Is there a preferred code formatter?

Reply:
No, please try to match the existing style, but no specific code formatter.